evm_body.yaml

```diff
@@ -2388,68 +2388,71 @@ alchemy_requestGasAndPaymasterAndData:
+                    description: Maximum fee per gas override
                   maxPriorityFeePerGas:
                     oneOf:
                       - type: string
                         description: 'Hex string'
                       - type: object
                         properties:
-                          percentage:
+                          multiplier:
                             type: number
                         required:
-                          - percentage
-                        description: 'Percentage value'
-                    description: Maximum priority fee per gas buffer
+                          - multiplier
+                        description: 'Multiplier value'
+                    description: Maximum priority fee per gas override
                   callGasLimit:
                     oneOf:
                       - type: string
                         description: 'Hex string'
                       - type: object
                         properties:
-                          percentage:
+                          multiplier:
                             type: number
                         required:
-                          - percentage
-                        description: 'Percentage value'
-                    description: Call gas limit buffer
+                          - multiplier
+                        description: 'Multiplier value'
+                    description: Call gas limit override
                   verificationGasLimit:
                     oneOf:
                       - type: string
                         description: 'Hex string'
                       - type: object
                         properties:
-                          percentage:
+                          multiplier:
                             type: number
                         required:
-                          - percentage
-                        description: 'Percentage value'
-                    description: Verification gas limit buffer
+                          - multiplier
+                        description: 'Multiplier value'
+                    description: Verification gas limit override
                   preVerificationGas:
                     oneOf:
                       - type: string
                         description: 'Hex string'
                       - type: object
                         properties:
-                          percentage:
+                          multiplier:
                             type: number
                         required:
-                          - percentage
-                        description: 'Percentage value'
-                    description: Pre-verification gas buffer
+                          - multiplier
+                        description: 'Multiplier value'
+                    description: Pre-verification gas override
+                  paymasterAndData:
+                    type: string
```

Comment:
Just a string? What's an example?

Reply:
It's a hex string. Is that a type option?

Reply:
We should probably be explicit about that. Let me find an example.

Reply:
Okay, we're not very consistent about this. I guess it's fine as is.